docs/java-rest/high-level/licensing/post-start-basic.asciidoc

```diff
@@ -0,0 +1,55 @@
+[[java-rest-high-post-start-basic]]
```

Comment:
You may want to lift some of the wording from the *very* similar PR ive linked here, https://github.com/elastic/elasticsearch/blob/07554dda3b4bb2e2871d913f00e3cead0cedeeb4/docs/java-rest/high-level/licensing/start-trial.asciidoc .